contracts/0.8.25/vaults/ValidatorConsolidationRequests.sol

```diff
@@ -80,68 +65,102 @@ contract ValidatorConsolidationRequests {
      *      | ----- public key (48 bytes) ----- || ----- public key (48 bytes) ----- | ...
      *
      * @param _refundRecipient The address to refund the excess consolidation fee to.
-     * @param _stakingVault The address of the staking vault contract.
+     * @param _dashboard The address of the dashboard contract.
      * @param _adjustmentIncrease The sum of the balances of the source validators to increase the rewards adjustment by.
      */
-    function addConsolidationRequests(
+    function addConsolidationRequestsAndIncreaseRewardsAdjustment(
```

Comment:
there is a way to optimize memory consumption by avoiding all memory prealloc and rather doing:
```python
temp_buf.preallocate(96 /* two keys capacity */)

for k1,k2 in keys:
   memcpy(temp_buf, k1, k2)
   system_contract.call(temp_buf)
```